### PR TITLE
WIP: dummy get_ipython()

### PR DESCRIPTION
**Pull request by tacaswell**
Currently fails inside of nslsii.configure_base which directly imports
get_ipython

### CODE REVIEW

**Comment by dmgav** [comment]
I think we should copy those changes to the latest version of QueueServer which I will also post as a draft.

Another question: should we use RE and db instantiated in nslsii? We can probably discard db, but what about RE? Is there any reason to keep it?

File: bluesky_queueserver/manager/profile_ops.py [hunk 1]
```diff
@@ -49,10 +49,33 @@ def load_profile_collection(path):
...
+        def __init__(self, user_ns):
+            self.user_ns = user_ns
+
+        def __call__(self):
+            return self
+
+        def register_magics(self, *args, **kwargs):
+            ...
+
+        def magics(
```
Comment (mrakitin) [inline bluesky_queueserver/manager/profile_ops.py:64]: This seems to be an incomplete method.

Comment (dmgav) [reply bluesky_queueserver/manager/profile_ops.py:64]: As I understand it's not a working code, but work in progress. There are some fundamental issues that needs to be resolved before it works.

**Comment by dmgav** [comment]
One of the problems that I see is that we can not modify `nspace` from within a module executed by `runpy` by passing reference to `nspace` to this module as an entry to `nspace` dictionary.

File: bluesky_queueserver/manager/profile_ops.py (hunk 1 above)
Comment (tacaswell) [reply bluesky_queueserver/manager/profile_ops.py:64]: :+1:

**Comment by tacaswell** [comment]
I think we either need to do one of:

 a) monkey patch `IPython` with the above dummy
 b) monkey patch nslsii to make `configure_base` a mostly empty function (suspect we should just keep the imports)
 c) change nslsii upstream so that if `get_ipython() is None` then do not do the things that need it
 d) make `configure_base` take `get_ipython` as an (optional) input

**Comment by dmgav** [comment]
The feature is fully implemented in https://github.com/bluesky/bluesky-queueserver/pull/57

**Pull request closed without merging**